src/types/Form.ts

```diff
@@ -0,0 +1,5 @@
+export interface Form<T> {
+  author: T;
+  email: T;
+  text: T;
+}
```

Comment:
You don't need to use generic here, as this form is specific, and you won't have any other form where the authot/email/text will be number or boolean